build fail looks like it might be related
```
======================================================================
ERROR: testImportNone (corehq.apps.importer.tests.ImporterTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/importer/tests.py", line 94, in testImportNone
    self.assertEqual('EXPIRED', res['error'])
KeyError: 'error'
```

--- a/corehq/apps/importer/tests.py
+++ b/corehq/apps/importer/tests.py
@@ -91,12 +91,12 @@ class ImporterTest(TestCase):
 
     def testImportNone(self):
         res = do_import(None, self._config(), self.domain)
-        self.assertEqual('EXPIRED', res['error'])
+        self.assertEqual('EXPIRED', res['errors'])
         self.assertEqual(0, len(get_case_ids_in_domain(self.domain)))
 
     def testImporterErrors(self):
         res = do_import(MockExcelFile(has_errors=True), self._config(), self.domain)
-        self.assertEqual('HAS_ERRORS', res['error'])
+        self.assertEqual('HAS_ERRORS', res['errors'])
         self.assertEqual(0, len(get_case_ids_in_domain(self.domain)))
 
     def testImportBasic(self):

--- a/corehq/ex-submodules/soil/exceptions.py
+++ b/corehq/ex-submodules/soil/exceptions.py
@@ -1,6 +1,6 @@
 
 
 class TaskFailedError(Exception):
-    def __init__(self, errors, *args, **kwargs):
+    def __init__(self, errors=None, *args, **kwargs):
         self.errors = errors
         super(TaskFailedError, self).__init__(*args, **kwargs)

